debug/notify.c

```diff
@@ -174,24 +174,31 @@ static void notify_dump_color(struct NotifyCallback *nc)
 
   const char *color = NULL;
   const char *scope = "";
+  const char *op = "";
 
-  if (nc->event_subtype == MT_COLOR_MAX)
+  if (ev_c->color == MT_COLOR_MAX)
     color = "ALL";
 
   if (!color)
-    color = mutt_map_get_name(nc->event_subtype, Fields);
+    color = mutt_map_get_name(ev_c->color, Fields);
 
   if (!color)
   {
-    color = mutt_map_get_name(nc->event_subtype, ComposeFields);
+    color = mutt_map_get_name(ev_c->color, ComposeFields);
     scope = "compose ";
   }
 
   if (!color)
     color = "UNKNOWN";
 
-  mutt_debug(LL_DEBUG1, "\tColor: %s %s%s (%d)\n", ev_c->set ? "set" : "reset",
-             scope, color, nc->event_subtype);
+  if (nc->event_subtype == NT_COLOR_SET)
+    op = "set";
+  else if (nc->event_subtype == NT_COLOR_RESET)
+    op = "reset";
+  else
+    op = "unknown";
+
+  mutt_debug(LL_DEBUG1, "\tColor: %s %s%s (%d)\n", op, scope, color, ev_c->color);
```

Comment:
I think I would push the conditional inside the `mutt_debug` parameter:
```c
mutt_debug(LL_DEBUG1, "\tColor: %s %s%s (%d)\n",
           (nc->event_subtype == NT_COLOR_SET)
           ? "set"
           : (nc->event_subtype == NT_COLOR_RESET)
             ? "reset"
             : "unknown",
           scope, color, ev_c->color);
```

Reply:
done